{{cookiecutter.project_name}}/pyproject.toml

```diff
@@ -78,6 +78,10 @@ pytest = "^7.1.0"
 pytest-cov = "^3.0.0"
 pytest-xdist = "^3.3.1"
 vulture = "{{ cookiecutter._vulture_version }}"
+{%- if cookiecutter.__project_type=="agent_plugin" %}
+monkey-agentpluginapi = "{{ cookiecutter._monkeyagentpluginapi_version }}"
+monkey-types = "{{ cookiecutter._monkeytypes_version }}"
+{%- endif %}
```

Comment:
NIT: It would be nice if these were in alphabetical order with the rest of the dev dependencies.